src/analysis/special_functions/gaussian.lean

```diff
@@ -98,109 +119,237 @@ end
+    rw [div_re, of_real_im, of_real_re, zero_mul, zero_div, add_zero],
+    exact div_pos (mul_pos pi_pos hb) (norm_sq_pos.mpr (nv hb)), },
+  { -- RHS doesn't vanish
+    rw [ne.def, cpow_eq_zero_iff, not_and_distrib],
+    exact or.inl (div_ne_zero (of_real_ne_zero.mpr pi_ne_zero) (nv hc)) },
+end
+
+/- The Gaussian integral on the half-line, `∫ x in Ioi 0, exp (-b * x^2)`, for complex `b`. -/
+lemma integral_gaussian_complex_Ioi {b : ℂ} (hb : 0 < re b) :
+  ∫ x:ℝ in Ioi 0, cexp (-b * x^2) = (π / b) ^ (1 / 2 : ℂ) / 2 :=
+begin
+  have full_integral := integral_gaussian_complex hb,
   have : measurable_set (Ioi (0:ℝ)) := measurable_set_Ioi,
-  rw [←integral_add_compl this (integrable_exp_neg_mul_sq hb), compl_Ioi] at full_integral,
-  suffices : ∫ x in Iic 0, exp (-b * x^2) = ∫ x in Ioi 0, exp (-b * x^2),
+  rw [←integral_add_compl this (integrable_cexp_neg_mul_sq hb), compl_Ioi] at full_integral,
+  suffices : ∫ x:ℝ in Iic 0, cexp (-b * x^2) = ∫ x:ℝ in Ioi 0, cexp (-b * x^2),
   { rw [this, ←mul_two] at full_integral,
     rwa eq_div_iff, exact two_ne_zero },
-  have : ∀ (c : ℝ), ∫ x in 0 .. c, exp (-b * x^2) = ∫ x in -c .. 0, exp (-b * x^2),
+  have : ∀ (c : ℝ), ∫ x in 0 .. c, cexp (-b * x^2) = ∫ x in -c .. 0, cexp (-b * x^2),
   { intro c,
-    have := @interval_integral.integral_comp_sub_left _ _ _ _ 0 c (λ x, exp(-b * x^2)) 0,
+    have := @interval_integral.integral_comp_sub_left _ _ _ _ 0 c (λ x, cexp (-b * x^2)) 0,
     simpa [zero_sub, neg_sq, neg_zero] using this },
   have t1 := interval_integral_tendsto_integral_Ioi _
-     ((integrable_exp_neg_mul_sq hb).integrable_on) tendsto_id,
-  have t2 : tendsto (λ c:ℝ, ∫ x in 0 .. c, exp (-b * x^2)) at_top (𝓝 ∫ x in Iic 0, exp (-b * x^2)),
+     ((integrable_cexp_neg_mul_sq hb).integrable_on) tendsto_id,
+  have t2 : tendsto (λ c:ℝ, ∫ x:ℝ in 0..c,
+    cexp (-b * x^2)) at_top (𝓝 ∫ x:ℝ in Iic 0, cexp (-b * x^2)),
   { simp_rw this,
     refine interval_integral_tendsto_integral_Iic _ _ tendsto_neg_at_top_at_bot,
-    apply (integrable_exp_neg_mul_sq hb).integrable_on },
+    apply (integrable_cexp_neg_mul_sq hb).integrable_on },
   exact tendsto_nhds_unique t2 t1,
 end
 
+/- The Gaussian integral on the half-line, `∫ x in Ioi 0, exp (-b * x^2)`, for real `b`. -/
+lemma integral_gaussian_Ioi (b : ℝ) : ∫ x in Ioi 0, exp (-b * x^2) = sqrt (π / b) / 2 :=
+begin
+  rcases le_or_lt b 0 with hb|hb,
+  { rw [integral_undef, sqrt_eq_zero_of_nonpos, zero_div],
+    exact div_nonpos_of_nonneg_of_nonpos pi_pos.le hb,
+    rwa [←integrable_on, integrable_on_Ioi_exp_neg_mul_sq_iff, not_lt] },
+  rw [←of_real_inj, ←integral_of_real],
+  convert integral_gaussian_complex_Ioi (by rwa of_real_re : 0 < (b:ℂ).re),
+  { ext1 x, simp, },
+  { rw [sqrt_eq_rpow, ←of_real_div, of_real_div, of_real_cpow],
+    norm_num,
+    exact (div_pos pi_pos hb).le, }
+end
+
 namespace complex
 
 /-- The special-value formula `Γ(1/2) = √π`, which is equivalent to the Gaussian integral. -/
```

Comment:
Can't this proof be shortened now that you the value of the complex integral?

Reply:
Sadly it doesn't seem to help much (the fiddly bit of `Gamma_one_half_eq` is about handling the `x^(1 / 2 - 1)` term, not the exponential).